cli/util/yaml.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ProjectConfig struct {
```

Comment:
I was thinking this would head to both. I can imagine having a rig-wide config that allows controlling default behaviors on both project config and non-project config.

This is ProjectConfig here because the struct started as P, and my refactoring only took the intermediate step. This should probably just be interface in this file, and in a new project-config.go or in the existing project.go remap it to the more explicit data structure.